### PR TITLE
Kick off فارسی (Persian) localization

### PR DESCRIPTION
**Pull request by sattarfeizollahibarough**
This is the Persian Language documentation PR.

Thank you.

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -213,4 +213,11 @@ aliases:
     - butuzov
     - idvoretskyi
     - MaxymVlasov
-    - Potapy4
\ No newline at end of file
+    - Potapy4
+  sig-docs-fa-owners: # Admins for Persian content
+    - sattarfeizollahibarough
+    - kasramp
+  sig-docs-uk-reviews: # PR reviews for Persian content
```
[the remark this answers is not in the document]
Comment (sattarfeizollahibarough) [reply OWNERS_ALIASES]: @rlenferink thank you for your consideration, I have updated the file.

File: content/fa/OWNERS [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+# Teams and members are visible at https://github.com/orgs/kubernetes/teams.
+
+reviewers:
+- sig-docs-fa-reviews
+
+approvers:
+- sig-docs-fa-owners
+
+labels:
+- language/fa
```
[the remark this answers is not in the document]
Comment (sattarfeizollahibarough) [reply content/fa/OWNERS:13]: @rlenferink thank you for your comment, This file has been updated and The PR regarding test-infra is add.
https://github.com/kubernetes/test-infra/pull/18369

**Comment by sattarfeizollahibarough** [comment]
I signed it

**Comment by sftim** [comment]
/retitle Add فارسی (Persian) localization

(hope that's OK)

**Comment by sftim** [comment]
I think this is a work-in-progress (it needs more work before it is ready to review); is that right @sattarfeizollahibarough?

**Comment by sattarfeizollahibarough** [comment]
@sftim Yes, That's right and thank you for your contribution. We are working on the translation right now, however, It's mentioned that on the first step we should PR for minimum requirements.

**Comment by sattarfeizollahibarough** [comment]
@sftim Yes you are right, working on a new branch is much more better idea. I would be appreciated if you could do that. Thank you in advance.

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -60,6 +60,13 @@ aliases:
     - alexbrand
     # glo-pena
     - electrocucaracha
+  sig-docs-fa-owners: # Admins for Persian content
+    - sattarfeizollahibarough
+    - kasramp
+  sig-docs-fa-reviews: # PR reviews for Persian content
+    - sattarfeizollahibarough
+    - kasramp
+    - FaezeAzimi
```
Comment (zacharysarah) [inline OWNERS_ALIASES:69]: Excellent, thanks for adding this. Two things:

1. Alphabetize each block
2. Add the same alphabetized team blocks to https://github.com/kubernetes/org/blob/master/config/kubernetes/sig-docs/teams.yaml#L74.

File: content/fa/OWNERS [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+# This is the localization project for Persian.
+# Teams and members are visible at https://github.com/orgs/kubernetes/teams.
+
```
Comment (zacharysarah) [inline content/fa/OWNERS:5]: If it's helpful to your team, you can also add this text in Persian. 👍

File: OWNERS_ALIASES (hunk 3 above)
Comment (sattarfeizollahibarough) [reply OWNERS_ALIASES]: The requested changes have been made.
https://github.com/kubernetes/org/pull/2051

File: content/fa/OWNERS (hunk 4 above)
Comment (sattarfeizollahibarough) [reply content/fa/OWNERS:5]: @zacharysarah your comments were constructive.

**Comment by kasramp** [comment]
Hi @zacharysarah and @sftim. I've been discussing about K8s Persian translations with @sattarfeizollahibarough offline for sometimes. 
To proceed with it further (and become a trusted org member), shall I open an issue similar to https://github.com/kubernetes/org/issues/2058 and tag you guys as there sponsors?

Thanks

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> Hi @zacharysarah and @sftim. I've been discussing about K8s Persian translations with @sattarfeizollahibarough offline for sometimes.
> To proceed with it further (and become a trusted org member), shall I open an issue similar to [kubernetes/org#2058](https://github.com/kubernetes/org/issues/2058) and tag you guys as there sponsors?
> 
> Thanks

Hi @kasramp, Yes you should submit the request. I can be the first sponsor. And We appreciate if @sftim can be the second sponsor.

**Comment by sftim** [comment]
@sattarfeizollahibarough you can now change the base branch for this PR to [_dev-1.18-fa.1_](https://github.com/kubernetes/website/tree/dev-1.18-fa.1) and we can look at getting this merged. Then it will make sense to open a new long lived PR for merging _dev-1.18-fa.1_ into _master_, because you can then use the long-lived PR to preview the Persian localization.

I suggest that you initially omit the owner files to get started, those can arrive in a later PR.

**Comment by sftim** [comment]
I'd be happy to merge this now @sattarfeizollahibarough - you could remove the [WIP] marker (or I can).

You can also visit https://github.com/kubernetes/website/pull/new/dev-1.18-fa.1 _right now_ to create a WIP pull request for merging the _dev-1.18-fa.1_ branch into _master_. As soon as this PR merges, the preview for that not-yet-created PR will update automatically.

File: content/fa/docs/concepts/overview/what-is-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
+---
+title: کوبرنتیز چیست?
+description: >
+  کوبرنتیز یک پلتفرم قابل انتقال، قابل گسترش و کد باز است که برای مدیریت اپلیکیشنهای مبتنی بر کانتینر استفاده می‌شود و با استفاده از این پلتفرم می‌توان خودکارسازی و پیکره‌بندری توصیقی را به سادگی انجام داد.
+content_type: مفاهیم
```
Comment (rlenferink) [inline content/fa/docs/concepts/overview/what-is-kubernetes.md:5]: On my phone right now so hard to check with other localizations. But shouldn’t the content_type be specified in English @sftim ?

Comment (sftim) [reply content/fa/docs/concepts/overview/what-is-kubernetes.md:5]: Yep, it's a keyword.
```suggestion
content_type: concept
```

**Comment by sftim** [comment]
Actually, @sattarfeizollahibarough, about:
> it will make sense to open a new long lived PR for merging dev-1.18-fa.1 into master, because you can then use the long-lived PR to preview the Persian localization.

can you open that _now_ and mark the PR as work-in-progress?

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> Actually, @sattarfeizollahibarough, about:
> 
> > it will make sense to open a new long lived PR for merging dev-1.18-fa.1 into master, because you can then use the long-lived PR to preview the Persian localization.
> 
> can you open that _now_ and mark the PR as work-in-progress?

Hi @sftim,
I tried to open a PR for merging dev-1.18-fa.1 into master, however, there is nothing to be merged and it is reasonable. Should I first approve this PR and then open the aforementioned PR?

Capture1


Or maybe you are saying, I should open a PR from sattarfeizollahibarogh:dev-1.18-fa.1 to kubernetes:master?

**Comment by sftim** [comment]
Oh of course, you can't open an empty PR (at least not via the web UI).
Let me see if this is OK to merge with just 1 owner.

**Comment by zacharysarah** [comment]
@sftim 
> Let me see if this is OK to merge with just 1 owner.

A localization needs at least two people in order to review and approve each other's PRs. I see that [OWNERS_ALIASES](https://github.com/kubernetes/website/pull/22565/files#diff-0de52d14f35759dfa67139f90c82bd7fR63-R66) only includes @sattarfeizollahibarough. 

@sattarfeizollahibarough 👋 Is there someone else you can collaborate with on Persian localization?

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> @sftim
> 
> > Let me see if this is OK to merge with just 1 owner.
> 
> A localization needs at least two people in order to review and approve each other's PRs. I see that [OWNERS_ALIASES](https://github.com/kubernetes/website/pull/22565/files#diff-0de52d14f35759dfa67139f90c82bd7fR63-R66) only includes @sattarfeizollahibarough.
> 
> @sattarfeizollahibarough 👋 Is there someone else you can collaborate with on Persian localization?

Hi @zacharysarah,
Happy to see you again and I hope you enjoyed your travel. Yes, there are two other cool guys @kasramp and @FaezeAzimi.

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough 

> Yes, there are two other cool guys @kasramp and @FaezeAzimi.

Excellent! Please add them to the OWNERS_ALIASES file in this repo. 

As a [reminder](https://github.com/kubernetes/org/pull/2051#issuecomment-661999251), both @kasramp and @faezeamimi need to become members of the Kubernetes org. When they are both org members, please reopen https://github.com/kubernetes/org/pull/2051#issuecomment-661999251 and add them to the Persian teams there as well.

> Happy to see you again and I hope you enjoyed your travel. 

good-thanks

**Comment by sattarfeizollahibarough** [comment]
> As a [reminder](https://github.com/kubernetes/org/pull/2051#issuecomment-661999251), both @kasramp and @FaezeAzimi need to become members of the Kubernetes org. When they are both org members, please reopen [kubernetes/org#2051 (comment)](https://github.com/kubernetes/org/pull/2051#issuecomment-661999251) and add them to the Persian teams there as well.

@zacharysarah I have done the OWNERS_ALIASES file. unfortunately, both @kasramp and @FaezeAzimi need two sponsors for joining Kubernetes ORG. Is it possible, they name you and someone else as a sponsor? if yes who?

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough 

>  Is it possible, they name you and someone else as a sponsor? if yes who?

I'm happy to sponsor them both. @sftim @jimangel @kbhawkey would you be willing?

**Comment by sftim** [comment]
I'm also happy to sponsor.

**Comment by sftim** [comment]
Hi. I'd also like to fix the commit message for commit `0bf95fbacdc5b8b52e59f676b9a36dc8435e798` as commit messages in PRs should, ideally, not include pull request numbers.

/hold
for that

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> Hi. I'd also like to fix the commit message for commit `0bf95fbacdc5b8b52e59f676b9a36dc8435e798` as commit messages in PRs should, ideally, not include pull request numbers.
> 
> /hold
> for that

@sftim 👋 is this issue resolved?

**Comment by sftim** [comment]
https://github.com/kubernetes/website/pull/22565/commits still lists a commit message that GitHub highlights as a reference to an issue or pull request

(look for the blue highlighting)

**Comment by sattarfeizollahibarough** [comment]
@sftim Hi,
I have resolved this issue by rolling back to the certain merge. When @kasramp and @FaezeAzimi trying to pull request to my repo we have the auto generated problem. I think at first we should merge this PR to the dev-1.18-fa.1 branch and then @kasramp and  @FaezeAzimi PR individually.

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough 👋 It looks like you've resolved the linking commit message. If you're ready to merge this PR and iterate, please squash your commits (currently I see 21 commits) into a single commit.

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> @sattarfeizollahibarough wave It looks like you've resolved the linking commit message. If you're ready to merge this PR and iterate, please squash your commits (currently I see 21 commits) into a single commit.
@zacharysarah :wave: 
Done.

File: config.toml [hunk 6]
```diff
@@ -445,3 +445,10 @@ contentDir = "content/uk"
 time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
+
+[languages.fa]
+title = "Kubernetes"
+description = "ارکستراسیون کانتینرها برای محیطهای عملیاتی"
+languageName = "Persian"
```
Comment (sftim) [inline config.toml:452]:
```suggestion
languageName = "فارسی"
```

File: content/fa/docs/concepts/overview/what-is-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+
+  کوبرنتیز یک پلتفرم قابل انتقال، قابل گسترش و کد باز است که برای مدیریت اپلیکیشن‌های مبتنی بر کانتینر استفاده می‌شود و با استفاده از این پلتفرم می‌توان خودکارسازی و پیکره‌بندی توصیقی را به سادگی انجام داد. کوبنتیز یک اکوسیستم بزرگ و به سرعت در حال رشد دارد. پشتیبانی از آن و ابزارهایش به صورت فراگیر در دسترس هستند.
+
+نام کوبنتیز از واژه‌ای یونانی به معنی سکان‌دار یا ناخدا نشات گرفته است. شرکت گوگل این پروژه را در سال 2014 به صورت کد باز تعریف کرد. در واقع کوبرنتیز حاصل [بیش از پانزده سال تجربه شرکت گوگل](/blog/2015/04/borg-predecessor-to-kubernetes/) در فراهم آوردن محیط عملیاتی شلوغ و پر ترافیک است که با بهترین ایده‌ها و به‌روش‌های جامعه ادغام شده است.
+
+## تاریخچه
+
+نگاهی به گذشته در پاسخ به این سوال که چرا کوبرنتیز بوجود آمد موثر خواهد بود.
+
+![Deployment evolution](/images/docs/Container_Evolution.svg)
```
Comment (sftim) [inline content/fa/docs/concepts/overview/what-is-kubernetes.md:26]: nit: you can localize the alt attribute

(OK to fix that later)

**Review by sftim — COMMENTED** [review]
Nice.

You can make this into a buildable branch by:
copying `content/en/_common-resources/` into `content/fa/_common-resources` (as is)
translating `content/en/case-studies/_index.html` to `content/fa/case-studies/_index.html`

**Comment by sftim** [comment]
I would also rebase this against _master_ to pick up the language direction metadata that's now in `config.toml`. For Persian, set `languagedirection = "rtl"`

(this doesn't make the site become right-to-left by itself, but it's a necessary change so we can consider customizing the rendering per localization as appropriate).

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> I would also rebase this against _master_ to pick up the language direction metadata that's now in `config.toml`. For Persian, set `languagedirection = "rtl"`
> 

:wave:  @sftim 
you mean in 'config.toml' file I add something like that?

[languages.fa]
title = "Kubernetes"
description = "ارکستراسیون کانتینرها برای محیطهای عملیاتی"
languageName = "فارسی"
contentDir = "content/fa"
weight = 15
languagedirection = "rtl"

as it is stated in hugo website (https://gohugo.io/content-management/multilingual/)

**Comment by sftim** [comment]
@sattarfeizollahibarough I'd like you to rebase this PR against master to pick up the languagedirection changes; something like:
```shell
git checkout master
git pull --ff-only # make sure this pulls from https://github.com/kubernetes/website
git checkout persian-language
git rebase master
# edit config.toml to add languagedirection rtl
git add -p config.toml
git commit --amend # Revise existing commit
git push --force-with-lease # this should be pushing to your fork
```

**Comment by sftim** [comment]
Hmm, actually we might need to bump _dev-1.18-fa.1_ before you can do that. I'll see if that's easy.

**Comment by sftim** [comment]
Done. Please try this instead:
```
git checkout dev-1.18-fa.1
git pull --ff-only # make sure this pulls from https://github.com/kubernetes/website
git checkout persian-language
git rebase dev-1.18-fa.1
# edit config.toml to add languagedirection rtl
git add -p config.toml
git commit --amend # Revise existing commit
git push --force-with-lease # this should be pushing to your fork
```

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough 
> you mean in 'config.toml' file I add something like that?
> languagedirection = "rtl"

Yes please. 👍

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> @sattarfeizollahibarough
> 
> > you mean in 'config.toml' file I add something like that?
> > languagedirection = "rtl"
> 
> Yes please. +1

@zacharysarah :wave: 
Done.

**Comment by zacharysarah** [comment]
@sattarfeizollahibarough Thanks. 

@sftim @kbhawkey 👋 Any thoughts on why Netlify seems to have [abandoned the build](https://github.com/kubernetes/website/pull/22565#issuecomment-660318373) for this PR?

**Comment by sftim** [comment]
The preview is missing since https://github.com/kubernetes/website/pull/22565#event-3586265153

There's no previewing configured (yet) for _dev-1.18-fa.1_

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> The preview is missing since [#22565 (comment)](https://github.com/kubernetes/website/pull/22565#event-3586265153)
> 
> There's no previewing configured (yet) for _dev-1.18-fa.1_

@sftim :wave:  What is preview? And how should I config that?

**Comment by sftim** [comment]
There's nothing wrong with this PR; it's not getting an automatic Netlify preview because the SIG Docs team (me included) haven't yet set that up for Persian.

**Comment by sftim** [comment]
@FaezeAzimi / @kasramp are you happy to see these changes merge into the development branch _kubernetes:dev-1.18-fa.1 f_?

**Comment by kasramp** [comment]
Hi @sftim - it's a yes from my side. I've already reviewed the translations and they are good to merge :-)

**Comment by sftim** [comment]
@kasramp can you add /lgtm so it's a bit more official?

**Comment by FaezeAzimi** [comment]
Hello @sftim. Yes of course. Thank you.
/lgtm

**Comment by sftim** [comment]
I'd be happy to merge this. I can't review the Farsi / Persian, but I'm keen to see this move forward.
/approve

File: config.toml [hunk 8]
```diff
@@ -461,3 +461,11 @@ languagedirection = "ltr"
 time_format_blog = "02.01.2006"
 # A list of language codes to look for untranslated content, ordered from left to right.
 language_alternatives = ["en"]
+
```
Comment (kbhawkey) [inline config.toml:464]: At some point, the `fa` language param should be placed alphabetically into the list of languages?

**Comment by sftim** [comment]
What I might suggest @sattarfeizollahibarough is that I push a _manually rebased_ version of this PR, and then merge that.
Before I do I'll collect the appropriate approvals etc.

File: content/fa/_common-resources/images/blocks.svg [hunk 9]
```diff
@@ -0,0 +1 @@
...cls-1" x="90.22" y="166.085" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="89.99" y="8.775" width="33.6" height="33.6" rx="4.158"/><rect class="cls-3" x="89.99" y="48.1" width="33.6" height="33.6" rx="4.158"/><rect class="cls-3" x="49.914" y="8.775" width="33.6" height="33.6" rx="4.158"/><rect class="cls-3" x="10.24" y="86.925" width="33.6" height="33.6" rx="4.158"/><rect class="cls-3" x="50.23" y="166.085" width="33.6" height="33.6" rx="4.158"/><rect class="cls-3" x="130.21" y="126.825" width="33.6" height="33.6" rx="4.158"/><rect class="cls-3" x="170.2" y="166.085" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="49.914" y="48.1" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="50.21" y="86.925" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="130.15" y="86.925" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="170.2" y="126.825" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="10.24" y="126.825" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="130.21" y="166.085" width="33.6" height="33.6" rx="4.158"/><rect class="cls-2" x="90.22" y="126.825" width="33.6" height="33.6" rx="4.158"/></svg>
```
Comment (kbhawkey) [inline content/fa/_common-resources/images/blocks.svg:1]: I have not looked at the branch or preview. Do you need to commit the SVG files?

**Comment by sftim** [comment]
:warning: this PR changes the Japanese localization - is that intended?

File: content/fa/docs/setup/_index.md [hunk 10]
```diff
@@ -0,0 +1,26 @@
+---
+no_issue: true
+title: setup
+main_menu: true
```
Comment (kbhawkey) [inline content/fa/docs/setup/_index.md:4]: nit: Does this front matter match the front matter in `content/en/docs/setup/_index.md`?

**Comment by kbhawkey** [comment]
> 
> 
> ⚠️ this PR changes the Japanese localization - is that intended?

@sattarfeizollahibarough , ☝️ , Is there a reason to delete the three `ja` files? If not, are you able to remove these changes from this PR?

**Comment by kbhawkey** [comment]
> 
> 
> @sattarfeizollahibarough Thanks.
> 
> @sftim @kbhawkey 👋 Any thoughts on why Netlify seems to have [abandoned the build](https://github.com/kubernetes/website/pull/22565#issuecomment-660318373) for this PR?

Hi. I just saw this comment. I'd like to see a build of this branch before merging. What is missing?

**Comment by kbhawkey** [comment]
Is the membership pending:
https://github.com/kubernetes/org/pull/2051

**Comment by sftim** [comment]
Netlify doesn't build previews for that target branch.

**Comment by kbhawkey** [comment]
> 
> 
> @sattarfeizollahibarough
> 
> > Is it possible, they name you and someone else as a sponsor? if yes who?
> 
Sorry for the late response. Yes, I can sponsor.

File: config.toml (hunk 8 above)
Comment (sattarfeizollahibarough) [reply config.toml]: I've changed the order.

**Comment by kbhawkey** [comment]
> 
> 
> Netlify doesn't build previews for that target branch.

Yes, but should it?

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> > @sattarfeizollahibarough
> > > Is it possible, they name you and someone else as a sponsor? if yes who?
> 
> Sorry for the late response. Yes, I can sponsor.

@kbhawkey Thank you for you kindness, @sftim and @zacharysarah sponsored me and I have already joint to the Kubernetes members. Any other membership is required?

**Comment by sattarfeizollahibarough** [comment]
> warning this PR changes the Japanese localization - is that intended?

Nope! I think it's my mistake. I will fix that.

**Comment by sattarfeizollahibarough** [comment]
> > warning this PR changes the Japanese localization - is that intended?
> 
It is kind of weird. :confused: I couldn't find the given files ( docker.md, mirror-pod.md , and static-pod.md ) in none of the branches :thinking:

File: config.toml [hunk 11]
```diff
@@ -420,14 +432,14 @@ title = "Kubernetes"
...
-weight = 12
+weight = 13
 languagedirection = "ltr"
 
 [languages.ru]
 title = "Kubernetes"
 description = "Первоклассная оркестрация контейнеров"
 languageName = "Русский"
-weight = 12
+weight = 13
```
Comment (kbhawkey) [inline config.toml:442]: ?

Comment (sattarfeizollahibarough) [reply config.toml:442]: Currently in master repository of Kubernetes the Vietnamese and Russian language have the same weight 11!!!

**Comment by kbhawkey** [comment]
> 
> 
> > > warning this PR changes the Japanese localization - is that intended?
> 
> It is kind of weird. 😕 I couldn't find the given files ( docker.md, mirror-pod.md , and static-pod.md ) in none of the branches 🤔

Is this what you are looking for:
https://github.com/kubernetes/website/blob/master/content/ja/docs/reference/glossary/docker.md

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> > > > warning this PR changes the Japanese localization - is that intended?
> > 
> > 
> > It is kind of weird. 😕 I couldn't find the given files ( docker.md, mirror-pod.md , and static-pod.md ) in none of the branches 🤔
> 
> Is this what you are looking for:
> https://github.com/kubernetes/website/blob/master/content/ja/docs/reference/glossary/docker.md

Yep, But I supposed those files should be in dev-1.18 or dev-118-ja branches, which obviously they are not! 
Should I copy those from the master branch to my branch?

**Comment by sftim** [comment]
> Should I copy those from the master branch to my branch?
This is where I think a manual rebase + squash could be helpful. I can help with that if needed.

**Comment by sattarfeizollahibarough** [comment]
> 
> 
> > Should I copy those from the master branch to my branch?
> 
> This is where I think a manual rebase + squash could be helpful. I can help with that if needed.

Thank you @sftim could you help me to do that?

**Comment by sftim** [comment]
> Thank you @sftim could you help me to do that?

Let's get the (virtual) paperwork in place first, then I can use my access as tech lead to take care of it all.

File: config.toml (hunk 11 above)
Comment (kbhawkey) [reply config.toml:442]: Sorry for the confusion. I do see that there is duplication and did not realize the ordering of the
weights.

**Comment by sftim** [comment]
@sattarfeizollahibarough I have force-pushed _dev-1.18-fa.1_ as [discussed](https://github.com/kubernetes/website/pull/22565#issuecomment-693500572).

If you visit https://github.com/kubernetes/website/pull/new/dev-1.18-fa.1 you can open a pull request (I suggest you mark it as work-in-progress) to merge that into _master_

**Comment by lmktfy** [comment]
Also see https://github.com/kubernetes/website/pull/51600

**Pull request merged**